### ISSUE TITLE
Other syntaxes

### ISSUE THREAD

**Issue by rafapaulin** [SublimeLinter/SublimeLinter-jshint#94]
Is there, by any chance, a way to SublimeLinter-jshint work with other syntaxes? (I am using [EcmaScript-Sublime](https://github.com/bathos/Ecmascript-Sublime)). Thanks

**Comment by travellingprog** [comment]
@rafapaulin I'm not a contributor to SublimeLinter-jshint, but I had a similar issue where jshint was never running because I'm using the "JavaScript (Babel)" syntax from the [Babel](https://github.com/babel/babel-sublime) package.

I found that Sublime Linter's settings provide an easy fix with the [syntax_map](http://www.sublimelinter.com/en/latest/global_settings.html#syntax-map) setting. Basically, all I had to do was open the Sublime Linter - User settings, and I added the key-value entry `"javascript (babel)": "javascript"` to the syntax_map object.

So now my full syntax_map object is:
```
"syntax_map": {
    "html (django)": "html",
    "html (rails)": "html",
    "html 5": "html",
    "javascript (babel)": "javascript",
    "php": "html",
    "python django": "python"
}
```

So this tells Sublime Linter "if there's a file with a 'javascript (babel)' syntax, use the same linter that would be applied for a file with a 'javascript' syntax". You probably need to do something similar for compatibility with the EcmaScript-Sublime package.
